nvfuser/__init__.py

```diff
@@ -10,6 +10,8 @@
 # This is needed when libnvfuser.so is patched and doesn't have the pytorch library location available.
 sys.path.append(os.path.join(os.path.dirname(torch.__file__), "lib"))
 
+cmake_prefix_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'nvfuser', 'cmake')
```

Comment:
nitpick: shouldn't this just be `os.path.join(os.path.dirname(__file__), 'cmake')`?

Reply:
without the `nvfuser`, the path is `/home/rspring/miniconda3/lib/python3.9/site-packages/nvfuser-0.0.8+git486678e-py3.9-linux-x86_64.egg/`